src/main/java/org/mockbukkit/mockbukkit/profile/PlayerProfileMock.java

```diff
@@ -275,7 +275,7 @@ public boolean equals(Object obj)
 	}
 
 	@Override
-	@SuppressWarnings("MethodDoesntCallSuperMethod")
+	@SuppressWarnings({"java:S2975", "MethodDoesntCallSuperMethod"})
 	public org.bukkit.profile.@NotNull PlayerProfile clone()
```

Comment:
Shouldn't this not null come from jetbrains? Just a strange direct reference here.